src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,51 @@
+package duke;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+import duke.command.AddDeadlineCommand;
+import duke.command.AddToDoCommand;
+import duke.command.ListTasksCommand;
+import duke.parser.Parser;
+import duke.task.Deadline;
+import duke.task.ToDo;
+
+
+
+public class ParserTest {
```

Comment:
Really appreciate the extensive testing you applied to your code by going above and beyond the minimal requirements